docs/api/wrapper/isVueInstance.md

```diff
@@ -1,7 +1,11 @@
 ## isVueInstance
 
-::: warning
+::: warning Deprecation warning
 `isVueInstance` is deprecated and will be removed in future releases.
+
+Most of the times, tests relying on `isVueInstance` provide little to no-value so you could simply remove them.
```

Comment:
Technically those are only valid in shallowMount environments, to assert dynamic components via `<component :is=""`, nothing much else.... I had to use it probably once or twice.

The proposed solution does the job :)